lib/confluence/aggressive_removal_data.es6.js

```diff
@@ -0,0 +1,63 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+'use strict';
+
+foam.CLASS({
+  name: 'AggressiveRemovalData',
+  package: 'org.chromium.apis.web',
+  documentation: `This class contains a browser and its
+    number of failure to ship APIs.`,
+  ids: ['browserName', 'date'],
+  properties: [
+    {
+      name: 'browserName',
+      class: 'String',
+      documentation: 'Name of the Browser',
+      required: true,
+      final: true,
+    },
+    {
+      name: 'browserOneYearAgo',
+      documentation: `An Browser object that is released just before
+        date - 1 year time.`,
+      required: true,
+      final: true,
+    },
+    {
+      name: 'browserVersion',
+      class: 'String',
+      documentation: 'Version of this browser.',
+      required: true,
+      final: true,
+    },
+    {
+      name: 'date',
+      class: 'Date',
+      documentation: 'The date at which this metric is computed.',
+      required: true,
+      final: true,
+    },
+    {
+      name: 'numAggressiveRemoval',
+      class: 'Int',
+      documentation: 'The number of aggressive-removal APIs.',
+      required: true,
+      final: true,
+    },
+    {
+      name: 'prevReleaseBrowsers',
```

Comment:
Not urgent: We can always look this up based on other info in this object. I don't think it needs to be modeled here.